backend/analytics_server/mhq/service/incidents/sync/etl_git_incidents_handler.py

```diff
@@ -173,6 +173,7 @@ def _process_revert_pr_incident(
             resolved_date=revert_pr_map.revert_pr.state_changed_at,
             assigned_to=revert_pr_map.revert_pr.author,
             assignees=[revert_pr_map.revert_pr.author],
+            url=revert_pr_map.revert_pr.url,
```

Comment:
Shouldn't this be original pr url?

Reply:
The call can be taken, I kept the revert PR because it generally has links to the original PR. And hence user can access both. Let me know if you think otherwise.

Reply:
You can derive the same inference by going to an original PR, keeps the model consistent.